Source/Core/Core/HLE/HLE.h

```diff
@@ -34,7 +34,8 @@ u32 UnPatch(const std::string& patchName);
 bool UnPatch(u32 addr, const std::string& name = {});
 void Execute(u32 _CurrentPC, u32 _Instruction);
 
-u32 GetFunctionIndex(u32 em_address);
+u32 GetFunctionIndex(u32 address);
+u32 GetFirstFunctionIndex(u32 address);
```

Comment:
You may want to consider documenting the behavioral differences/intent between the two.